static/workspace.html

```diff
@@ -19,6 +19,7 @@
       src="../node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"
       rel="script"
     ></script>
+    <script src="https://cdn.jsdelivr.net/npm/chart.js@3.6.0/dist/chart.min.js"></script>
```

Comment:
please add your dependency as package to npm instead of adding it to the header :)